app/javascript/src/dashboard.js

```diff
@@ -116,24 +71,141 @@ $('document').ready(() => {
     }
   )
 
+  const handleAjaxError = e => {
+    if (e.status === 401) {
+      location.reload();
+    } else {
+      console.log(e);
+      if (e.responseJSON && e.responseJSON.error) {
+        alert(e.responseJSON.error);
+      } else if (e.response.statusText) {
+        alert(`Unable to complete your request: ${e.response.statusText}`);
+      } else {
+        alert("Unable to complete your request.");
+      }
+    }
+  };
+
   // Enable all data tables on dashboard but only filter on volunteers table
+  const editSupervisorPath = id => `/supervisors/${id}/edit`;
+  const editVolunteerPath = id => `/volunteers/${id}/edit`;
+  const casaCasePath = id => `/casa_cases/${id}`;
   var volunteersTable = $('table#volunteers').DataTable({
     autoWidth: false,
     stateSave: false,
-    columnDefs: [
+    columns: [
       {
-        targets: [1],
+        name: "display_name",
+        render: (data, type, row, meta) => {
+          return `
+            <a href="${editVolunteerPath(row.id)}">
+              ${row.display_name || row.email}
+              ${row.made_contact_with_all_cases_in_days === 'false'
+                ? `🕐 <i class="fa fa-question-circle" aria-hidden="true" data-toggle="tooltip" title="Has at least one case with no contact in 14 days"></i>`
```

Comment:
This tooltip is unclickable because it is included in the volunteers link. This needs to get fixed. (was it always broken like that? Yes I know I'm the one who wrote it, sorry y'all 😅)